src/game.js

```diff
@@ -0,0 +1,60 @@
+function Game() {
+  this.rolls = [];
+  this.score = 0;
+  this.frame = 1;
```

Comment:
At the moment, the code incrementally calculates the `frame` and `score` as rolls come in. Why might it be better for the `score` to be calculated whenever it's asked for (by making `score` a method)?